site/docs/latest/admin/http.md

```diff
@@ -0,0 +1,394 @@
+---
+title: BookKeeper http endpoints
```

Comment:
I would suggest changing "http endpoint" to "Admin REST API", which would be more accurate.

Reply:
thanks